packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -155,7 +155,7 @@ const ProofOfIdentityContainer = ({
     }, [createVerificationConfig, previous_account_status, account_status]);
 
     const { needs_poa, is_unwelcome, allow_document_upload } = verification_status;
-    const is_rejected = identity_status_key === onfido_status_codes.rejected;
+    const is_rejected = identity_status_key === onfido_status_codes.rejected || onfido_status_codes.suspected;
```

Comment:
```suggestion
    const is_rejected = identity_status_key === onfido_status_codes.rejected || identity_status_key === onfido_status_codes.suspected;
```

or you can use regex

Reply:
Hi @mustofa-binary, please have a look. For any new rejection status we just need to update the array, "rejectionStatus"